Source/WTF/wtf/text/StringView.h

```diff
@@ -230,13 +229,13 @@ class StringView final {
 
     void clear();
 
-    const void* m_characters { nullptr };
-    unsigned m_length { 0 };
-    bool m_is8Bit { true };
-
 #if CHECK_STRINGVIEW_LIFETIME
     UnderlyingString* m_underlyingString { nullptr };
 #endif
+
+    PackedPtr<const void> m_characters;
```

Comment:
Note that I also tried:
```
CompactPointerType<const void*, bool /* is8Bit */> m_characters;
unsigned m_length;
```

and 
```
const void* m_characters;
unsigned m_length : 31 { 0 };
unsigned m_is8Bit : 1 { true };
```

But only this approach with the `PackedPtr<>` managed to keep the size of `CSSParserToken` at 24.